packages/kbn-dependency-usage/src/dependency_graph/common/constants.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export const aggregationGroups = [
+  'x-pack/plugins',
+  'x-pack/packages',
+  'src/plugins',
+  'packages',
+  'src',
+  'x-pack/test',
+  'x-pack/test_serverless',
+];
+
+export const excludePaths = [
+  '(^|/)target($|/)',
+  '^kbn',
+  '^@kbn',
```

Comment:
why are these excluded from searches? don't we use kbn-* packages often as wrappers around dependencies?

Reply:
We are not interested in listing internal packages for the main output, if we want to know which external packages the kbn-* package has we can still run 
```
bash scripts/dependency_usage.sh -p packages/kbn-some-package
```